views/dashboard.ejs

```diff
@@ -0,0 +1,293 @@
+<head>
+  <title>Rose City Resource Admin Page</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css" />
+  <script>
+
+    async function fetchPgRows() {
+      let data = await fetch(window.location.href + '/pg-rows')
+        .catch(e => console.log(e))
```

Comment:
same comments about error handling.  Might be a good function to create so it can be reused in the catch blocks below.